esp/services/ws_topology/ws_topologyService.cpp

```diff
@@ -1373,6 +1373,9 @@ bool CWsTopologyEx::onTpClusterInfo(IEspContext &context, IEspTpClusterInfoReque
         if (conn)
         {
             Owned<IConstWUClusterInfo> clusterInfo = getTargetClusterInfo(req.getName());
+            if (clusterInfo == nullptr ||  clusterInfo.get() == nullptr)
```

Comment:
@rpastrana Do clusterInfo and clusterInfo.get() check the same thing?

Reply:
@wangkx the clusterInfo == nullptr portion might be superfluous, but it's testing the owned object, the get() tests the pointer.